metube/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: metube_metube_1
+      APP_PORT: 8081
+
+  metube:
+    image: ghcr.io/alexta69/metube:2023-11-16
```

Comment:
We need to include the multi-arch digest here so that we lock in the image for this version. In this case, it is:
```
ghcr.io/alexta69/metube:2023-11-16@sha256:566057b972b34b339e5ace77f44883852a4623b5961f84f24f33d337ab4a42ab
```